geometry_msgs/PoseWithCovarianceStamped initPos
geometry_msgs/PoseStamped goalPos
nav_msgs/OccupancyGrid map
std_msgs/Bool processCostMap
nav_msgs/OccupancyGrid costMap
---
nav_msgs/Path path
